src/httpClient.ts

```diff
@@ -44,6 +44,34 @@ export default class httpClient {
         }
     }
 
+    async getEsriStyles(url: string) {
+
+        const res = await fetch(url);
```

Comment:
`httpGet` already has lots of this boilerplate. maybe we can modify it to not always tack on `process.env.REACT_APP_ROUTE` so we can use it here as well

Reply:
Yea, I think this is a good idea. I'll separate out the current `httpget` so we aren't repeating